package.json

```diff
@@ -85,6 +85,7 @@
     "start": "node ./script/start.js",
     "test": "node ./script/spec-runner.js",
     "tsc": "tsc",
+    "yarn": "npx yarn@$(node -p 'require(\"./script/lib/utils\").YARN_VERSION')",
```

Comment:
does this work on windows?

Reply:
Doesn't need to 😄 We only use it on MacOS + Linux.  Appveyor doesn't need to call `yarn` manually anywhere.

Would be nice if it worked on windows too though, that would require a wrapping JS script 🤷‍♂ I can do that if you want?